src/modules/Builder/features/ActivitySettings/ScoresAndReports/ScoreContent/ScoreContent.utils.ts

```diff
@@ -93,3 +96,43 @@ export const getDefaultConditionalValue = (scoreId: string) => ({
   match: ConditionalLogicMatch.All,
   conditions: [{}],
 });
+
+export const getIsScoreIdVariable = (score: ActivitySettingsScore) => {
+  const { id } = score;
+  let isVariable = false;
+
+  const isMessageIncludeScoreId = (showMessage: boolean, message?: string) => {
```

Comment:
could you move this function outside of another function, no need to initialize it each time